src/DynamoCore/DynamoCore.csproj

```diff
@@ -30,6 +30,8 @@
     <PackageReference Include="System.Drawing.Common" Version="6.0.0" />
   </ItemGroup>
   <ItemGroup Label="Common dependencies">
+    <PackageReference Include="Autodesk.IDSDK" Version="1.1.8" />
```

Comment:
This IDSDK version was updated recently to 1.2.4. Sync with latest master before merging.